calibration/DOSTAM/CGINS-DOSTAM-00364__20180216.csv

```diff
@@ -1,3 +1,4 @@
-serial,name,value,notes
-364,CC_calibration_date,1.518739200000E+12,sensor calibration date (milliseconds since 1970-01-01); source_file DOSTA-M_Optode-4831_SN_364_Calibration_2018-02-16.pdf
-
+serial,name,value,notes
+364,CC_calibration_date,1.518739200000E+12,sensor calibration date (milliseconds since 1970-01-01); source_file DOSTA-M_Optode-4831_SN_364_Calibration_2018-02-16.pdf
+
```

Comment:
Looks like there is an extra line in this file. You'll probably want to remove it.

Reply:
okay. blank line removed.